user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md

```diff
@@ -56,6 +53,20 @@ For each DMA that should be connected, verify whether the DMA is operational and
 
    - If the page does not load correctly, this indicates a problem with the DMA host. Restart the DMA to resolve this. You can do so using the [DataMiner Taskbar Utility](xref:DataMiner_Taskbar_Utility).
 
+## Check the status of the CloudGateway DxM
+
+If the connection status is shown as red on [dataminer.services](https://dataminer.services/), validate that at least one CloudGateway instance is running in the cluster and that it can establish a connection towards dataminer.services:
+
+1. On the DMA servers, go to the Windows *Services* configuration window.
+
+1. Check whether the *DataMiner CloudGateway* service has the status *Running*.
+
+1. If the status is not *Running*, right-click the service and select *Start*.
+
+If there is a CloudGateway instance running, but the dataminer.services connection is still not valid, validate whether your network complies with the requirements for connecting to dataminer.services using the *ConnectionTester.exe* tool from the folder `Program files\Skyline Communications\Dataminer CloudGateway\`. More information on the requirements can be found 
```

Comment:
Something is still missing here. Could you add the missing info?

Reply:
Done